CodeStarterCamp_Week2/main.swift

```diff
@@ -1,12 +1,38 @@
-//
-//  main.swift
-//  CodeStarterCamp_Week2
-//
-//  Created by yagom.
-//  Copyright © yagom academy. All rights reserved.
-//
-
 import Foundation
 
-print("Hello, World!")
+// 로또 번호 생성 함수
+
+func makeLottoNums() -> Set<Int> {
+    var lotto: Set<Int> = []
+    
+    while lotto.count < 6 {
+        let randomInt: Int = Int.random(in: 1...45)
+        lotto.insert(randomInt)
+    }
+    
+    return lotto
+}
+
+// 로또 번호 당첨 확인 함수
+
+func checkMyLotto(myNumbers: [Int], realNumbers: Set<Int>) -> Void {
+    
+    let setMyNumbers: Set<Int> = Set(myNumbers)
+    
+    let intersectionLotto: Set<Int> = setMyNumbers.intersection(realNumbers)
+    
+    if intersectionLotto.count == 0 {
+        print("아쉽지만 겹치는 번호가 없습니다")
+    } else {
+        let matchLotto: [String] = intersectionLotto.map{String($0)}
```

Comment:
```suggestion
        let matchLotto: [String] = intersectionLotto.map(String.init)
```
요렇게도 쓸 수 있는데 저는 이게 더 깔끔해서 선호하는 편입니다~

아니라면, 
```
let matchLotto: [String] = intersectionLotto.map { String($0) }
```
요렇게 중괄호 앞뒤로는 띄워주시면 좋을 듯 해요~